src/components/totals/GoalProgress/index.js

```diff
@@ -49,6 +51,8 @@ module.exports = React.createClass({
     var offsetWidth = numeral(progress).format('0%');
     var style = { width: offsetWidth || '100%' };
 
+    if (!progress) { return null; }
```

Comment:
You don't have to return anything here.  To be more consise, you could just update the return statement below to `return progress && (`